docs/source/markdown/podman-completion.1.md

```diff
@@ -4,10 +4,10 @@
 podman\-completion - Generate shell completion scripts
 
 ## SYNOPSIS
-**podman completion** [*options*] *bash*|*zsh*|*fish*
+**podman completion** [*options*] *bash*|*zsh*|*fish*|*powershell*
 
 ## DESCRIPTION
-The completion command allows you to generate shell completion scripts. Supported shells are **bash**, **zsh** and **fish**.
+The completion command allows you to generate shell completion scripts. Supported shells are **bash**, **zsh**, **fish** and **powershell**.
```

Comment:
I know this is pre-existing, but we try to avoid you, we, our, etc in man pages.  Suggest:
```suggestion
The completion command generates shell completion scripts for a variety of shells. Supported shells are **bash**, **zsh**, **fish** and **powershell**.
```